packages/ts/react-crud/test/autogrid.spec.tsx

```diff
@@ -141,6 +151,66 @@ describe('@hilla/react-crud', () => {
       expect(getBodyCellContent(grid, 0, 1).innerText).to.equal('Love');
     });
 
+    describe('multi-sort', () => {
+      let grid: GridElement;
+      let service: CrudService<Person> & HasTestInfo;
+
+      function TestAutoGridWithMultiSort(customProps: Partial<AutoGridProps<Person>>) {
+        return (
+          <AutoGrid
+            service={personService()}
+            model={PersonModel}
+            noHeaderFilters
+            multiSort
+            multiSortPriority="append"
+            {...customProps}
+          ></AutoGrid>
+        );
+      }
+
+      beforeEach(async () => {
+        service = personService();
+        const result = render(<TestAutoGridWithMultiSort service={service} />);
+        await reactRender();
+        grid = getGrid(result);
+      });
+
+      it('sorts according to first column by default', () => {
+        expect(service.lastSort).to.eql({
+          orders: [{ property: 'firstName', direction: Direction.ASC, ignoreCase: false }],
+        });
+        expect(getSortOrder(grid)).to.eql([{ property: 'firstName', direction: Direction.ASC }]);
+      });
+
+      it('sorts by multiple columns', async () => {
+        sortGrid(grid, 'lastName', 'asc');
+        await reactRender();
+        expect(service.lastSort).to.eql({
+          orders: [
+            { property: 'firstName', direction: Direction.ASC, ignoreCase: false },
+            { property: 'lastName', direction: Direction.ASC, ignoreCase: false },
+          ],
+        });
+        expect(getSortOrder(grid)).to.eql([
+          { property: 'firstName', direction: Direction.ASC },
+          { property: 'lastName', direction: Direction.ASC },
+        ]);
+
+        sortGrid(grid, 'lastName', 'desc');
+        await reactRender();
+        expect(service.lastSort).to.eql({
+          orders: [
+            { property: 'firstName', direction: Direction.ASC, ignoreCase: false },
+            { property: 'lastName', direction: Direction.DESC, ignoreCase: false },
+          ],
+        });
+        expect(getSortOrder(grid)).to.eql([
+          { property: 'firstName', direction: Direction.ASC },
+          { property: 'lastName', direction: Direction.DESC },
+        ]);
+      });
+    });
+
```

Comment:
Could we have a test to make sure when triggering the sort for `lastName` for the third time, it is excluded from the list of sorter columns?

Reply:
The sort function actually just sets the direction programmatically, it doesn't use clicks:
https://github.com/vaadin/hilla/blob/07cf6f3bf17bae2b03579ee15e59dbda0eb28842/packages/ts/react-crud/test/grid-test-helpers.ts#L66-L72

Reply:
But still, removing the direction is a code path that should be tested, good point. Will add a test.

Reply:
Done